microprofile/microprofile-config-1.3/pom.xml

```diff
@@ -49,12 +49,23 @@
     </dependencies>
 
     <build>
-       <resources>
-         <resource>
-           <directory>src/main/resources</directory>
-           <filtering>true</filtering>
-         </resource>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+            </resource>
         </resources>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M3</version>
```

Comment:
Documentation says that `<environmentVariables>` are supported since version 2.1.3. Why set a version?